package.json

```diff
@@ -14,7 +14,9 @@
         "check:all": "yarn format:check && yarn lint:check",
         "fix:all": "yarn format:fix && yarn lint:fix",
         "prepare": "husky install",
-        "test": "jest --coverage"
+        "test": "jest --coverage",
+        "lint": "eslint .",
+        "lint-fix": "eslint . --fix"
```

Comment:
Why?

Reply:
This tells me exactly where I need to fix the linting, file and line now with the description,
It's optional

Reply:
modify the already written command